.github/workflows/build.yml

```diff
@@ -13,6 +13,8 @@ env:
   # We had a problem with GitHub setting quay expiration label also during
   # merge to main, so we just set meaningless value as a workaround.
   EXPIRATION_LABEL: ${{ github.event_name == 'push' && 'quipucords.source=github' || 'quay.expires-after=3d' }}
```

Comment:
I know it's unrelated, but can we bump this up to 5 days ?

Reply:
I like it! I updated this to 5 in the latest commit.